_posts/2016-05-27-floss.md

```diff
@@ -0,0 +1,79 @@
+
+---
+layout: post
+title: Open Source Software
+date: 2016-05-27 16:58:30 +530
+author: Harsh Narang
+website: home.iitk.ac.in/~harshn/
+category: FLOSS
+tags:
+- open source
+- FLOSS
+- linux
+- gnu
+- gnu/linux
+image:
+	url:
+credits:
+- https://en.wikipedia.org/wiki/Free_and_open-source_software
+- https://opensource.org/licenses
+-
+---
+
+#Introduction#
+
+Free Libre and open-source software (FLOSS) is computer software that can be classified as both free software and open-source software. That is, anyone is freely licensed to use, copy, study, and change the software in any way, and the source code is openly shared so that people are encouraged to voluntarily improve the design of the software. Free, open-source operating systems such as GNU/Linux are widely utilized today, powering millions of servers, desktops, smartphones (e.g. Android), and other devices.
+
+#History#
+
+It is common knowledge, from the 1950's to the 70's, the common trend for developers and users was to freely share source code with one another, and used to modify the code for tailoring it for their needs.
+
+However, in the 70's and early 80's, the evolving software industry began to directly sell software, which usually was in the form of binary copies, thereby preventing users to reverse engineer the software. In 1980, the copyright law in the States was extended to sotware programs.
+
+In 1983, Richard Stallman at the MIT AI Laboratory, announced the *GNU Project*. Along with the GNU OS development, the *Free Software Foundation a.k.a. FSF* was founded in 1985. The [GNU Manifesto][] contained the GNU philosophy of Free Software definition and "copyleft" ideas.
+[GNU Manifesto]: "https://en.wikipedia.org/wiki/GNU_Manifesto"
+
+The Linux kernel was released as a freely modifiable open source software in 1991, which was relicensed under the GNU GPL in 1992.
+FreeBSD and NetBSD were released as free software in 1993. In 1995, the Apache Web Server (A patchy web server, there is sure history behind its name :P) was released.
+
+Then came a revolutionary moment in 1998. The Netscape Corporation released its famous Netscape Communicator Suite as free software. (I am sure you guys must have heard Firefox and ThunderBird. Yeah, that is the descendant of the Netscape; however, another story, another time).
+
+Netscape saw business potential in collaborating and sharing source code, however the FSF and its ideals were not not appealing enough to them. Hence, they decided to rebrand the movement; coining the famous "open source" and the [Open Source Initiative][] was established in February, 1998. In 2001, the European Commission introduced the term FLOSS, which has been in both popular and official use since 2004.
```

Comment:
Why is `Open Source Initiative` link empty? The line below is redundant, remove it, and inside the para use this:
```
[Open Source](https://opensource.org)
```
In accordance with markdown syntax.